### PR TITLE
Add HarmonyOS library

### PR DESCRIPTION
**Pull request by sohamvg**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,30 +1,37 @@
...
-        google()
-        jcenter()
-        maven { url "https://jitpack.io" }
+        maven {
+            url 'https://repo.huaweicloud.com/repository/maven/'
+        }
+        maven {
+            url 'https://developer.huawei.com/repo/'
+        }
+		jcenter()
```
Comment (kanaksony) [inline build.gradle]: Is jcenter() repo required in build.gradle?

File: build.gradle [hunk 2]
```diff
@@ -1,30 +1,37 @@
...
     repositories {
-        google()
-        jcenter()
+        maven {
+            url 'https://repo.huaweicloud.com/repository/maven/'
+        }
+        maven {
+            url 'https://developer.huawei.com/repo/'
+        }
+		jcenter()
```
Comment (kanaksony) [inline build.gradle]: jcenter()?

File: entry/src/main/config.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      "target": 5,
+      "releaseType": "Release"
+    }
+  },
+  "deviceConfig": {},
+  "module": {
+    "package": "com.example.hassanusman.circulardialogs",
+    "name": ".MyApplication",
+    "deviceType": [
+      "phone"
```
Comment (kanaksony) [inline entry/src/main/config.json]: Add other device types in config.json like Tablet and fix issues find with same.

**Review by kanaksony — CHANGES_REQUESTED** [review]
Code not friendly for different screen resolutions. 
Proper indentation, extra and required spacing, code reformating required.

File: build.gradle (hunk 1 above)
Comment (sohamvg) [reply build.gradle]: Removed in commit 5693b42057797e7071ebb39d5c531d4eef3453bd.

File: build.gradle (hunk 2 above)
Comment (sohamvg) [reply build.gradle]: Removed in commit 5693b42057797e7071ebb39d5c531d4eef3453bd.

File: entry/src/main/config.json (hunk 3 above)
Comment (sohamvg) [reply entry/src/main/config.json]: Added tablet, tv in commit 5693b42057797e7071ebb39d5c531d4eef3453bd.

**Review by kanaksony — APPROVED** [review]
Looks good.

**Pull request merged**